modules/mini/src/main/java/org/apache/fluo/mini/MiniFluoImpl.java

```diff
@@ -214,7 +214,7 @@ public static void main(String[] args) {
         System.exit(-1);
       }
       try (MiniFluo mini = new MiniFluoImpl(config)) {
-        log.info("MiniFluo is running");
+        log.info("MiniFluo is running " + mini);
```

Comment:
The toString may not be meaningful at the moment, but it did resolve the compiler warning about an unreferenced resource inside a try-catch block. I initially tried to make it `while (mini != null)`, but findbugs didn't like that... it detected it as a possible unintentional infinite loop (vs. the current code, which is clearly an intentional infinite loop).

I can update the PR to remove the main method if it's not needed at all.